### PR TITLE
Remove conditions that are always true/false

### PR DESCRIPTION
**Pull request by Bond-009**
Part of #581

### CODE REVIEW

File: Emby.Drawing/SkiaEncoder.cs [hunk 1]
```diff
@@ -270,17 +270,10 @@ internal static SKBitmap Decode(string path, bool forceCleanBitmap, IFileSystem
                     // create the bitmap
                     var bitmap = new SKBitmap(codec.Info.Width, codec.Info.Height, !requiresTransparencyHack);
 
-                    if (bitmap != null)
-                    {
-                        // decode
-                        codec.GetPixels(bitmap.Info, bitmap.GetPixels());
+                    // decode
+                    codec.GetPixels(bitmap.Info, bitmap.GetPixels());
```
Comment (JustAMan) [inline Emby.Drawing/SkiaEncoder.cs]: am I missing something, or shouldn't result of call to `codec.GetPixels()` be assigned somewhere?

File: Emby.Server.Implementations/Library/UserManager.cs [hunk 2]
```diff
@@ -463,26 +460,26 @@ private void UpdateInvalidLoginAttemptCount(User user, int newValue)
             {
                 user.Policy.InvalidLoginAttemptCount = newValue;
 
-                var maxCount = user.Policy.IsAdministrator ?
-                    3 :
-                    5;
+                var maxCount = user.Policy.IsAdministrator ? 3 : 5;
 
+                // TODO: Fix
+                /*
```
Comment (JustAMan) [inline Emby.Server.Implementations/Library/UserManager.cs]: why commenting? pls explain

File: Emby.Drawing/SkiaEncoder.cs (hunk 1 above)
Comment (JustAMan) [reply Emby.Drawing/SkiaEncoder.cs]: Wait, what does GetPixels modify? Is it void?

File: Emby.Server.Implementations/Library/UserManager.cs (hunk 2 above)
Comment (JustAMan) [reply Emby.Server.Implementations/Library/UserManager.cs]: I mean, why did you decide to disable the code?

Comment (Bond-009) [reply Emby.Server.Implementations/Library/UserManager.cs]: Oh, it wasn't functional. It never executed.

File: Emby.Drawing/SkiaEncoder.cs (hunk 1 above)
Comment (Bond-009) [reply Emby.Drawing/SkiaEncoder.cs]: No.
`Returns SkiaSharp.SKCodecResult.Success on success, or another value explaining the type of failure.`

**Pull request merged**